readthedocs/doc_builder/python_environments.py

```diff
@@ -193,12 +193,12 @@ def install_core_requirements(self):
 
         # Use conda for requirements it packages
         requirements = [
-            'sphinx==1.3.5',
+            'sphinx==1.5.3',
             'Pygments==2.2.0',
-            'docutils==0.12',
+            'docutils==0.13.1',
             'mock',
             'pillow>=3.0.0',
-            'sphinx_rtd_theme==0.1.7',
+            'sphinx-rtd-theme<0.3',
```

Comment:
The latter is more consistent because `_` is replaced by `-`: http://stackoverflow.com/questions/19097057/pip-e-no-magic-underscore-to-dash-replacement